### PR TITLE
Release day fixes

### PR DESCRIPTION
**Pull request by RadiantFlash7**
Release day fixes!
Event disables/added stuff,
Tons of map issues fixed.

### CODE REVIEW

**Comment by Repede** [comment]
Every edited lines in a non _yw file should have a //YW tag in front so we know it's our changes.

File: code/modules/events/spider_migration.dm [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/datum/event/spider_migration
```
Comment (Repede) [inline code/modules/events/spider_migration.dm]: Add _yw to end of file

File: code/game/turfs/simulated/wall_types.dm [hunk 2]
```diff
@@ -255,6 +255,11 @@
 	hard_corner = 1
 	stripe_color = "#00FF00"
 
+/turf/simulated/shuttle/wall/voidcraft/hard_corner/red
```
Comment (Repede) [inline code/game/turfs/simulated/wall_types.dm]: add //YW Addition at beginning of the turf, or put it in a new file called wall_typed_yw.dm

File: code/modules/events/event_container.dm [hunk 3]
```diff
@@ -134,13 +134,13 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Brand Intelligence",/datum/event/brand_intelligence,20, 	list(ASSIGNMENT_JANITOR = 25),	1),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Camera Damage",		/datum/event/camera_damage,		20, 	list(ASSIGNMENT_ENGINEER = 20)),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Economic News",		/datum/event/economic_event,	300),
-		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Lost Carp",			/datum/event/carp_migration, 	20, 	list(ASSIGNMENT_SECURITY = 10), 1),
+		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Lost Spiders",		/datum/event/spider_migration, 	20, 	list(ASSIGNMENT_SECURITY = 10), 1),
```
Comment (Repede) [inline code/modules/events/event_container.dm]: Add //YW To beginning of every changed lines so we know which ones are from us.

File: code/modules/mob/living/simple_mob/subtypes/animal/giant_spider/frost.dm [hunk 4]
```diff
@@ -35,3 +35,6 @@
 /mob/living/simple_mob/animal/giant_spider/frost/sif
 	desc = "Icy and blue, it makes you shudder to look at it. This one has brilliant blue eyes. \
 	It isn't native to Sif."
+
+/mob/living/simple_mob/animal/giant_spider/frost/event
```
Comment (Repede) [inline code/modules/mob/living/simple_mob/subtypes/animal/giant_spider/frost.dm]: Add //YW addition to top of this line. Or put this new animal in a new file called frost_yw.dm

**Comment by RadiantFlash7** [comment]
I think that fixed, Maybe?

**Pull request merged**